class TaErradoError(Exception):
    pass


def testar():
    raise TaErradoError('TÃ¡ errado')

try:
    testar()
except TaErradoError as error:
    print(f'Erro: {error}')